ext/simplexml/simplexml.stub.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+/** @return SimpleXMLElement */
+function simplexml_load_file(string $filename, string $class_name = 'SimpleXMLElement', int $options = 0, string $ns = null, bool $is_prefix = false) {}
+
+/** @return SimpleXMLElement */
+function simplexml_load_string(string $data, string $class_name = 'SimpleXMLElement', int $options = 0, string $ns = null, bool $is_prefix = false) {}
+
+/** @return SimpleXMLElement */
+function simplexml_import_dom(DOMNode $node, string $class_name = 'SimpleXMLElement') {}
+
+class SimpleXMLElement {
+    /** @return array|false */
+    public function xpath(string $path) {}
+
+    /** @return bool */
+    public function registerXPathNamespace(string $prefix, string $ns) {}
+
+    /** @return string|bool */
+    public function asXML(string $filename = null) {}
```

Comment:
We can declare explicit that this argument accepts `null`:

```suggestion
    public function asXML(?string $filename = null) {}
```

The same suggestion applies to the next.